spec/support/git_spec_helpers.rb

```diff
@@ -23,6 +23,41 @@ def repo(options = {})
     end
   end
 
+  # Creates a local repository in 'local-repo' folder with a remote repository
+  # which points to 'some-repo' folder. The current working directory is set to
+  # the local repository's root directory.
+  def with_remote_repo
+    repo do
+      touch 'update-me'
+      `echo update >> update-me`
+      touch 'delete-me'
+      `echo delete >> delete-me`
+      `git add . 2>&1 > #{File::NULL}`
+      `git commit -m "Initial commit" 2>&1 > #{File::NULL}`
+
+      Dir.chdir '..' do
+        remote_url = "file://#{File.expand_path('some-repo')}"
```

Comment:
We should not hard code `some-repo` here (that assumes we know how the `repo` helper is implemented), but obtain it from the name of the repo directory that was created.

Reply:
My thought was to create a repo with a remote with some default files. Your comment does make sense. It could easily misinterpreted and it is not generic enough for re-use.